proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -99,6 +99,13 @@ service Query {
       option (google.api.http).get =
     "/interchain_security/ccv/provider/params";
     }
+
+  // QueryOldestUnconfirmVsc returns the send timestamp of the oldest unconfirmed VSCPacket for a given chainID
+  rpc QueryOldestUnconfirmVsc(QueryOldestUnconfirmVscRequest)
```

Comment:
Why not `QueryOldestUnconfirmedVsc`? Similarly for `QueryOldestUnconfirmVscRequest` and `QueryOldestUnconfirmVscResponse`.